.vib/kafka/goss/goss.yaml

```
@@ -0,0 +1,27 @@
+command:
+  create-kafka-topic:
+    exec: /opt/bitnami/kafka/bin/kafka-topics.sh --create --topic quickstart-events$RANDOM --bootstrap-server localhost:9092 
```

Actually, you could use:

```suggestion
    exec: kafka-topics.sh --create --topic quickstart-events$RANDOM --bootstrap-server localhost:9092 
```
and that would additionally check if `kafka-topics.sh` is in the PATH (which should be).